app.vue

```diff
@@ -1,56 +1,139 @@
+				/>
+				<PreviewCard
+					class="-rotate-12 xl:translate-y-1/3"
+					:content="contentCard2"
+					main-color="#3b82f6"
+					secondary-color="#bfdbfe"
+				/>
+				<PreviewCard
+					class="rotate-12 xl:-translate-x-1/3"
+					:content="contentCard3"
+					main-color="#d946ef"
+					secondary-color="#f5d0fe"
+				/>
+			</div>
+		</header>
+
+		<section
+			id="highlight"
+			class="mx-auto mt-20 max-w-7xl px-8 lg:mt-52"
+		>
+			<SectionTitle
+				title="Das Open Source Invoice Tool für Freelancer und kleine Unternehmen"
+			/>
+			<FeatureHighlight
+				eyebrow="Leichtes Rechnung erstellen"
+				title="Rechungen erstellen leicht gemacht"
+				content="Unser Rechnungsprogramm ermöglicht es Ihnen, schnell und effektiv
+            Rechnungen zu generieren und sofort an Kunden zu senden. Dies
+            beschleunigt den Zahlungseingang und verbessert Ihren Cashflow."
+				image-src="https://placehold.co/500x400"
+			/>
+			<FeatureHighlight
+				eyebrow="Leichtes Rechnung erstellen"
+				title="Rechungen erstellen leicht gemacht"
+				content="Unser Rechnungsprogramm ermöglicht es Ihnen, schnell und effektiv
+            Rechnungen zu generieren und sofort an Kunden zu senden. Dies
+            beschleunigt den Zahlungseingang und verbessert Ihren Cashflow."
+				image-src="https://placehold.co/500x400"
+				reverse
+			/>
+		</section>
+
+		<section
+			id="highlight2"
+			class="mx-auto mt-20 max-w-7xl px-4"
+		>
+			<SectionTitle
+				title="Das Open Source Invoice Tool für Freelancer und kleine Unternehmen"
+			/>
+			<FeatureHighlight
+				eyebrow="Leichtes Rechnung erstellen"
+				title="Rechungen erstellen leicht gemacht"
+				content="Unser Rechnungsprogramm ermöglicht es Ihnen, schnell und effektiv
+            Rechnungen zu generieren und sofort an Kunden zu senden. Dies
+            beschleunigt den Zahlungseingang und verbessert Ihren Cashflow."
+				image-src="https://placehold.co/500x400"
+			/>
+		</section>
+
+		<PromoSection />
+
+		<PricingSection
+			id="pricing"
+			title="Das Open Source Invoice Tool für Freelancer und kleine Unternehmen"
+			:pricing-options="pricingOptions"
+		/>
 	</div>
 </template>
 
-<style scoped>
-body {
-  background: #00091B;
-  color: #fff;
-}
-
-@keyframes fadeIn {
-  from { top: 20%; opacity: 0; }
-  to { opacity: 1; }
-}
+<script setup lang="ts">
+import type { PreviewCardContent } from '~/components/PreviewCard.vue'
+import PreviewCard from '~/components/PreviewCard.vue'
+import AppNavigation from '~/components/AppNavigation.vue'
+import PageTeaser from '~/components/PageTeaser.vue'
+import HeaderBackground from '~/components/HeaderBackground.vue'
+import FeatureHighlight from '~/components/FeatureHighlight.vue'
+import PromoSection from '~/components/PromoSection.vue'
 
-@-webkit-keyframes fadeIn {
-  from { top: 20%; opacity: 0; }
-  to { opacity: 1; }
+const contentCard1: PreviewCardContent = {
+	text: 'Hubspot API implementation',
+	value1: '29.640€',
+	value2: '38.000€',
```

Comment:
would be more prescise by naming the variables

maybe `totalRequired` `currentAmount` or `total` and `current`